mopidy_beetslocal/library.py

```diff
@@ -136,8 +136,9 @@ def browse(self, uri):
         elif level == "album":
             for track in self._browse_track(query):
                 result.append(Ref.track(
-                    uri="beetslocal:track:%s:%s" % (track.id,
-                                                    track.path.decode('utf8')),
+                    uri="beetslocal:track:%s:%s" % (
+                        track.id,
+                        uriencode(track.path.decode('utf8'), '/')),
```

Comment:
``` python
>>> uritools.uriencode(b'æøå')
'%C3%A6%C3%B8%C3%A5'
>>> uritools.uriencode('æøå')
'%C3%A6%C3%B8%C3%A5'
```

Thus, the `.decode('utf-8')` part is redundant. The decode-to-Unicode part might actually throw away data if the path isn't originally in UTF-8 or a subset like ASCII.